melodic/distribution.yaml

```diff
@@ -8270,6 +8270,21 @@ repositories:
       url: https://github.com/aws-robotics/monitoringmessages-ros1.git
       version: master
     status: maintained
+  ros_numpy:
+    doc:
+      type: git
+      url: https://github.com/eric-wieser/ros_numpy.git
+      version: master
+    release:
+      tags:
+        release: release/jade/{package}/{version}
```

Comment:
This isn't going to work.  Even though there have been no changes, you still need to re-bloom this for Melodic, which will create the melodic release branch.

Reply:
Alright, I changed it to melodic